syserol/tensor.py

```diff
@@ -127,6 +128,44 @@ def censored_lstsq(A: np.ndarray, B: np.ndarray, uniqueInfo) -> np.ndarray:
     return X.T
 
 
+def sigmoid(x: np.ndarray, P: np.ndarray):
+    """ Basic sigmoidal function """
+    x0, k = P
+    y = 1 / (1 + np.exp(-k*(x-x0)))
+    return y
+
+
+def F(P: np.ndarray, x: np.ndarray, y: np.ndarray):
```

Comment:
What is `F()`? A residuals function?

Reply:
Yes, a residuals function to minimize (if we use the sigmoidal method. now F has no purpose but I kept it in case we wanted to return to sigmoid)